### ISSUE TITLE
Outdated 

### ISSUE THREAD

**Issue by jsares** [ServiceStackApps/react-chat#1]
When using these instructions with latest version of Visual Studio 2017, lots of things are outdated and not working anymore : 
      - typings is deprecated, must use npm to install type definitions
      - project seems to originaly use TS1.8, when running in latest Typescript, current codebase shows 
        hundreds of warnings/ errors

**Comment by mythz** [comment]
Yeah such is the case with most JS frameworks. We've developed uptodate starting SPA project templates which use the latest JS fx versions (e.g. React 16) at:

 - [.NET Core 2.0 C# Templates](https://github.com/NetCoreTemplates)
 - [ASP.NET Framework C# Templates](https://github.com/NetFrameworkTemplates)

Both use the latest [v5 pre-release version of ServiceStack](https://forums.servicestack.net/t/myget-pre-release-packages-upgraded-to-v5/4749) which you can either get started with by cloning the GitHub repos in the above link or use our new [dotnet-new cli](https://forums.servicestack.net/t/myget-pre-release-packages-upgraded-to-v5/4749) tool which you can download with:

    $ npm install -g servicestack-cli

View available templates with:

    $ dotnet-new

And create either .NET Core 2.0 or .NET Framework Templates with:

    $ dotnet-new <template> <project-name>

E.g. you can create an ASP.NET React Single Page App Template with:

    $ dotnet-new react-spa-netfx ProjectName

Which downloads the [react-spa-netfx](https://github.com/NetFrameworkTemplates/react-spa-netfx) and uses your Project Name.
